### PR TITLE
[Schema] Enhance import design schema

### PR DESCRIPTION
**Pull request by Yashsharma1911**
**Description**

This PR fixes [#9145](https://github.com/meshery/meshery/issues/9145)

This PR implements conditional logic in the import design schema to display only 'URL Import' option exclusively when the user selects 'Helm Chart' as the design type, as file upload is not supported in this case. For other design types, the PR provides the option to choose between 'URL Import' and 'File Upload.' 

Additionally, I updated the title format from Title Case to Sentence case for improved consistency and enhanced field descriptions with additional helpful information and relevant documentation links.

Here is a video of the updated import modal

https://github.com/meshery/meshkit/assets/71271069/60c8bd3b-4c62-4657-83f0-076440447b1c

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Yashsharma1911** [comment]
Modal gets small on the selection of 'Helm Chart' as design type is issue for this I'll update this in Meshery with new PR

File: schemas/configuration/designImport.json [hunk 1]
```diff
@@ -1,79 +1,130 @@
...
-      "title": "Select the Upload Method",
-      "enum": [
-        "File Upload",
-        "URL Upload"
-      ],
-      "default": "File Upload",
-      "x-rjsf-grid-area" : "12"
+      "default": "Meshery Design",
+      "x-rjsf-grid-area": "12",
+      "description": "Select the type of design you are uploading. The 'Design Type' determines the format, structure, and content of the file you are uploading. Choose the appropriate design type that matches the nature of your file. Checkout https://docs.meshery.io/guides/meshery-design to learn more about designs"
```
Comment (theBeginner86) [inline schemas/configuration/designImport.json:22]: Will the link use our standard styling?

**Review by theBeginner86 — APPROVED** [review]
Looks good

File: schemas/configuration/designImport.json (hunk 1 above)
Comment (Yashsharma1911) [reply schemas/configuration/designImport.json:22]: Yes it is @theBeginner86 

Screenshot 2023-11-09 at 12 20 27 AM

**Comment by Yashsharma1911** [comment]
Thanks @theBeginner86 for reviewing, I'm moving forward here

**Comment by Yashsharma1911** [comment]
FYI @leecalcote

**Comment by leecalcote** [comment]
Yay. Looks good. “Url import” ✅

**Comment by leecalcote** [comment]
In general, if we can hyperlink text and not show the actual hyperlink, that’s better.

**Comment by Yashsharma1911** [comment]
Sounds reasonable I'll look

**Pull request merged**